profile/2020-09-01-hybrid.json

```diff
@@ -0,0 +1,612 @@
+        },
+        {
+          "resourceType": "locations/skus",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2019-07-01-preview/iothub.json"
+        },
+        {
+          "resourceType": "checkNameAvailability",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2019-07-01-preview/iothub.json"
+        },
+        {
+          "resourceType": "operations",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2019-07-01-preview/iothub.json"
+        },
+        {
+          "resourceType": "operationResults",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2019-07-01-preview/iothub.json"
+        },
+        {
+          "resourceType": "IotHubs",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2019-07-01-preview/iothub.json"
+        },
+        {
+          "resourceType": "backupProviders",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2019-07-01-preview/iothub.json"
+        },
+        {
+          "resourceType": "backupProviders/operationResults",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2019-07-01-preview/iothub.json"
+        }
+      ]
+    },
+    "microsoft.eventhubs": {
+      "2018-01-01-preview": [
+        {
+          "resourceType": "availableClusterRegions",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/AvailableClusterRegions-preview.json"
+        },
+        {
+          "resourceType": "clusters",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/Clusters-preview.json"
+        },
+        {
+          "resourceType": "namespaces",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/namespaces-preview.json"
+        }
+      ],
+      "2017-04-01": [
+        {
+          "resourceType": "checkNameAvailability",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json"
```

Comment:
There is no such file path, get 404.

Reply:
it looks like they renamed the file

Reply:
it should be individual file names under 
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01

instead of EventHub.json

Could you please unblock yourself with this change? I will make a PR to correct this